src/services/userAuthentication.js

```diff
@@ -51,6 +51,19 @@ class UserAuthentication {
     );
   }
 
+  claimUsersRewardBalance() {
+    const self=this;
+    return this.steemUser.claimRewardBalance(
+      self.username,
+      self.userInOurDb.steemBalance,
+      self.userInOurDb.sbdBalance,
+      self.userInOurDb.vestingBalance,
+      function(err, res) {
+        return err ? new Error(err) : self.userInOurDb;
+      }
```

Comment:
Let's use ES6 when possible, and we shouldn't wrap the `err` object we receive in another `Error` object.

```
(err, res) => err ? err : self.userInOurDb;
```